Lib/unittest/test/testmock/testpatch.py

```diff
@@ -105,6 +105,9 @@ def test():
         self.assertEqual(Something.attribute, sentinel.Original,
                          "patch not restored")
 
+    def test_patchobject_with_string_as_target(self):
+        with self.assertRaisesRegex(TypeError, f"'Something' must be the actual object to be patched, not a str"):
```

Comment:
[PEP 8](https://www.python.org/dev/peps/pep-0008/) states that all lines should be limited to 79 chars.

Also, you shouldn't prefix a string with `f` if there are no replacement fields. I believe a variable should be used here (so that `'Something'` isn't hardcoded twice).

Reply:
Agreed on not using `f`-strings here, but I think it's perfectly fine to repeat the use of `'Something'`. It's a fairly straight-forward test and adding in a variable for it doesn't provide much benefit.

Reply:
Do you want me to remove that f string and use simple string?